public/react-services/wz-agents.ts

```diff
@@ -30,3 +34,16 @@ export function hasAgentSupportModule(agent, component){
   const agentOSType = getAgentOSType(agent);
   return !(UnsupportedComponents[agentOSType].includes(component));
 };
+
+
+export async function getAuthorizedAgents() {
+  const agentsList: IApiResponse<{id: string}> = await WzRequest.apiReq('GET', `/agents`, {})
```

Comment:
I think we could need a loop to get all the authorized agents. The current endpoint has a limit of agents returned (`500`). This should be reviewed.